db/entities/attempt.go

```diff
@@ -16,12 +17,16 @@ type Attempt struct {
 	AttemptedAt   *types.Time   `json:"attempted_at" db:"attempted_at"`
 
 	ErrorCode *AttemptErrorCode `json:"error_code" db:"error_code"`
-	Request   *AttemptRequest   `json:"request" db:"request"`
-	Response  *AttemptResponse  `json:"response" db:"response"`
```

Comment:
Should use one struct(Attempt) in both PageAttempt and GetAttempt API, instead of deleting them and creating a new one AttemptWithDetail with many redundant fields.

Reply:
@webhookx-x  agree, i will keep them in `Attempt`, but they are no longer db columns, so the tag `db` should be deleted, and the `attempts` table also should remove them.

Reply:
And entities `Struct` keep fields that no longer columns in db will make some bad effects:
i tried to alter table `attempts` to drop `request` and `response`, and  the `travel` function still scans all the struct fields include `request` and `response`, even though they don't have tag `db`. 
It will make DAO functions failed.
Maybe `travel` support `db:"-"` that omit the field is a solution.

Reply:
And entities Struct keep fields that no longer columns in db will make some bad effects:
i tried to alter table attempts to drop request and response, and the travel function still scans all the struct fields include request and response, even though they don't have tag db.
It will make DAO functions failed.
Maybe support db:"-" that omit the field is a solution.